images/rootfs.yml.in

```diff
@@ -66,6 +66,9 @@ services:
      image: XENTOOLS_TAG
      cgroupsPath: /eve/services/xen-tools
      oomScoreAdj: -999
+   - name: vtune
```

Comment:
Let's drop it from the default builds for now and move it to a special profile (see how we're doing `rootfs-acrn.yml.in.patch` for example) -- the rest looks good @temerkhanov